sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/jms/AzureServiceBusJMSProperties.java

```diff
@@ -76,4 +83,77 @@ public void validate() {
             throw new IllegalArgumentException("'spring.jms.servicebus.pricing-tier' is not valid");
         }
     }
+
+    /**
+     * Properties to configure {@link org.springframework.jms.annotation.JmsListener} for
+     * {@link org.springframework.jms.config.AbstractJmsListenerContainerFactory}.
+     */
+    public static class Listener {
+
+        /**
+         * Whether the reply destination type is topic. Only works for the bean of topicJmsListenerContainerFactory.
+         */
+        private Boolean replyPubSubDomain;
+
+        /**
+         * Configure the {@link QosSettings} to use when sending a reply.
+         */
+        private QosSettings replyQosSettings;
+
+        /**
+         * Whether to make the subscription durable. Only works for the bean of topicJmsListenerContainerFactory.
+         */
+        private Boolean subscriptionDurable = Boolean.TRUE;
```

Comment:
This value will only take effects on the topic listener in our starter. But for JMS, it works on a common listener, so that's why its default value is false. 
And for us, to keep the legacy code for no breaking changes brought in and also we need a flag property for a topic listener container, I set its default value as true.